.github/pull_request_template.md

```diff
@@ -1,10 +1,19 @@
-Write a detailed description of your PR here.
+Please make sure to follow all the tasks in the checklist below. Then, delete this line and write a detailed description of your pull request here.
 
-Please follow the guidelines for PRs contained [here](docs/pull_requests.md). Checklist:
+<!--
+## Guidelines
 
-- [ ] Ensure that your PR is small, and implements one change.
-- [ ] Add unit tests for all functions that you introduced or modified.
-- [ ] Run PyCharm's code cleanup tools on your Python files.
-- [ ] Link the correct GitHub issue for tracking.
-- [ ] When merging your PR, replace the default merge message with a description of your PR,
-and if needed a motivation why that change was required.
+Please follow the guidelines for pull requests (PRs) in [CONTRIBUTING](/docs/contributing.md). Checklist:
+
+- Ensure that your PR is small, and implements one change
+- Give your PR title one of the prefixes ENH, BUG, STYLE, DOC, DEL to indicate what type of change that is (see [CONTRIBUTING](/docs/contributing.md))
+- Link the correct GitHub issue for tracking
+- Add unit tests for all functions that you introduced or modified
+- Run automatic code formatting / linting on all files ("Format Document" Shift-Alt-F in VSCode)
+- Ensure that documentation renders correctly in Sphinx by running `make html` in the `docs` folder
```

Comment:
I don't think there's a Makefile in the docs folder, is there?

Reply:
That's right. I copied this file from `hi-ml` and forgot to adapt this line. It should say `sphinx-docs`. But InnerEye docs seem to be less relevant, so I'll just remove the line. Thanks!